templates/frontOffice/default/account.html

```diff
@@ -60,7 +60,7 @@ <h4 class="panel-title">
                                     {if $CELLPHONE != ""}
                                         <span class="mobile">{$CELLPHONE}</span>
                                     {/if}
-                                    <span class="email">{mailto address="{$EMAIL}" encode="hex"}</span>
+                                    <span class="email">{mailto address=$EMAIL encode="hex"}</span>
```

Comment:
Since you wrote the CustomerCreateForm class. This data is checked by the form validator :

`->add("email", "email", array(
                "constraints" => array(
                    new Constraints\NotBlank(),
                    new Constraints\Email(),
                   ...
`